docs/docs/09-commands-and-tools/02-ide-support.md

```diff
@@ -208,6 +209,107 @@ If you still don't get syntax highlighting after it's installed, try running `:T
 vim.api.nvim_create_autocmd("BufEnter", { pattern = "*.templ", callback = function() vim.cmd("TSBufEnable highlight") end }) 
 ```
 
+### Minimal Config
+
+Minimal config with the following features (useful for debugging):
```

Comment:
Do you have time to explain what you need to do to use this config?

IIRC, you've got to install a plugin or two, and then run `nvim -u <config.lua>` or something?

Reply:
nothing more todo than just use the config in nvim (in the nvim appname folder).
also `-u` will not work here because you need to set the actual "home folder" (aka NVIM_APPNAME folder) this way lazy (and other nvim functionality) will actually work corretly

will add it now